### DISCUSSION TITLE
Can't fetch total commits while rank_icon=github

### DISCUSSION THREAD

**Discussion by StevenWanglolz** [DenverCoder1/github-readme-streak-stats#647] [Q&A] [answered]
As the title states, include_all_commits=true&rank_icon=github causes an issue and cannot fetch total commits. This is my code
```
<a href="https://github.com/anuraghazra/github-readme-stats">
  <img height=200 align="center" src="https://github-readme-stats.vercel.app/api?username=StevenWanglolz&include_all_commits=true&theme=dracula&rank_icon=github">
</a>
```
<a href="https://github.com/anuraghazra/github-readme-stats">
  <img height=200 align="center" src="https://github-readme-stats.vercel.app/api?username=StevenWanglolz&include_all_commits=true&theme=dracula&rank_icon=github">
</a>

**Comment by DenverCoder1** [comment, marked as answer]
This issue is for a different project

Your are probably looking for https://github.com/anuraghazra/github-readme-stats
